### PR TITLE
update(docs): update docs for context creation

### PR DESCRIPTION
**Pull request by frdomovic**
# update(docs): update docs for context creation

### CODE REVIEW

File: docs/03-getting-started/03-example-app.mdx [hunk 1]
```diff
@@ -9,10 +9,20 @@ you need to create new context where application will be installed.
...
+
+```bash title="Terminal"
+$ meroctl context create --application <app-id>
+```
+
+2. If application is not installed then create new context using command:
+
+```bash title="Terminal"
+$ meroctl context create --watch /path/to/app
+```
```
Comment (miraclx) [inline docs/03-getting-started/03-example-app.mdx:25]: this should be `meroctl app install /path/to/app` but app install doesn't support paths for now, just IDs from the registry

**Review by miraclx — APPROVED** [review]
aside what's mentioned (which is out of scope for this PR, though can be confusing) the rest seem fine to me

File: docs/03-getting-started/03-example-app.mdx (hunk 1 above)
Comment (frdomovic) [reply docs/03-getting-started/03-example-app.mdx:25]: Before I change this, could you verify this command first?
The goal of point 2. is to create new context not install new application. So as I see in implementation "meroctl context create --watch /path/to/app" first installs the app then starts the context. or should i just rewrite this like this:

```bash title="Terminal"
$ meroctl app install --application <app-id>
>> <app-id>
$ meroctl context create --application <app-id>
>> {}

Comment (miraclx) [reply docs/03-getting-started/03-example-app.mdx:25]:
```console
$ meroctl app install --path /path/to/app
>> <app-id>
$ meroctl context create --application <app-id>
>> <context-id>
```

Comment (frdomovic) [reply docs/03-getting-started/03-example-app.mdx]: updated

**Pull request merged**